.changeset/wicked-cups-tie.md

```diff
@@ -0,0 +1,8 @@
+---
+'docs': patch
+'angular-example': patch
+'@aws-amplify/ui-angular': patch
+'e2e': patch
```

Comment:
Should `docs`, `angular-example`, & `e2e` be ignored?

Reply:
Yeah, I'm divided on this. Note that this is what shows up by default when you click the "add changeset" link from the bot. This will be ignored on `changeset version` or `changeset publish` as per `.changeset/config.json`. 

I personally don't like the DX of having to remove these ignored packages manually, so I kept it as-is. Let's seee how this gets consumed in #596 and we can have a DX discussion based off that.

Reply:
Ahhhhhh https://github.com/aws-amplify/amplify-ui/runs/4109281658?check_suite_focus=true

> 🦋  error Error: Found mixed changeset wicked-cups-tie
🦋  error Found ignored packages: docs angular-example e2e
🦋  error Found not ignored packages: @aws-amplify/ui-angular

We do need to remove those unused packages. TIL! I'll just push something to main directly to fix that shortly.